src/adapters/netsuite/index.js

```diff
@@ -85,11 +86,32 @@ async function findContact({ user, authHeader, phoneNumber, overridingFormat })
                     })
                 }
             }
+            //For Customer search
+            const customerInfo = await axios.post(
+                `https://${user.hostname.split(".")[0]}.suitetalk.api.netsuite.com/services/rest/query/v1/suiteql`,
+                {
+                    q: `SELECT id,firstname,middlename,lastname FROM customer WHERE phone = ${numberToQuery}`
```

Comment:
quick check. Does netsuite just have one type of phone? Some CRMs have phone, mobile, workPhone etc.